lib/mountain_view/component.rb

```diff
@@ -2,6 +2,10 @@ module MountainView
   class Component
     attr_reader :name
 
+    def self.stub_example
+      File.read(File.expand_path("../example.yml", __FILE__))
+    end
```

Comment:
Sure no problem.

I will move it to an instance method, and see the possibility of making it an `erb` file. 
Thanks for the feedback @kitop